android/app/build.gradle

```diff
@@ -38,5 +38,5 @@ task javadoc(type: Javadoc) {
 }
 
 dependencies {
-    testImplementation 'junit:junit:4.12'
+    testImplementation "androidx.test.ext:junit:1.1.1"
```

Comment:
AndroidX is not a single library, but a huge set of libraries. This one is used for testing only, so it will not even be embedded in the result aar file.

Reply:
And again, this one is just one of the libraries for testing that contains only JUnit. There are other testing libraries in AndroidX, but they are not needed here.

Reply:
I'm just trying to make sure that developers can use the library without `AndroidX`. That's why I asked. OK then, it looks like it won't create a problem.